library/src/main/java/io/ona/kujaku/views/KujakuMapView.java

```diff
@@ -226,14 +233,14 @@ public void onClick(View v) {
                 buttonsLayout.setVisibility(VISIBLE);
 
                 if (useGPS) {
-                    enableAddPoint(true, null);
+                    enableAddPoint(true, (OnLocationChanged) null);
```

Comment:
Is it necessary to cast the null?

Reply:
Yeap

Reply:
It would be helpful if you provided more information on this.

Reply:
Different method signatures for the same method name. Try it out on your own IDE

Reply:
I understand that us case, this method is not overloaded though.

There is only one method with this signature.

The IDE shows that the casting is unnecessary.

Reply:
Fixed